vernac/indschemes.ml

```diff
@@ -460,12 +463,39 @@ let build_combined_scheme env schemes =
 	      ctx, ind, spec.mind_nrealargs
 	| _ -> ctx, destInd last, 0
   in
+  (* Return true iff the target sort is Prop *)
+  let is_sort_prop ty =
```

Comment:
I think this function can be simplified to `Retyping.get_sort_family_of ty == Sorts.InProp`.